packages/engine/Source/Core/ApproximateTerrainHeights.js

```diff
@@ -45,9 +45,15 @@ ApproximateTerrainHeights.initialize = function () {
   }
   initPromise = Resource.fetchJson(
     buildModuleUrl("Assets/approximateTerrainHeights.json")
-  ).then(function (json) {
-    ApproximateTerrainHeights._terrainHeights = json;
-  });
+  )
+    .then(function (json) {
+      ApproximateTerrainHeights._terrainHeights = json;
+    })
+    .catch(function (error) {
```

Comment:
So just to clarify, the promise was being rejected, but the returned value is `undefined`?

Reply:
I don't know what promise was rejected, or where `undefined` was thrown. ApproximateTerrainHeights was simply a guess, since it adds global state to so many test suites.

What is not clear to me is:
- Why throwing the error solves the problem
- Why the thrown error is not reported in the browser

Reply:
The error may be reported in the browser in the console.